+ community code of conduct (#10518)
ethereum/ethereum-org-website: src/content/community/code-of-conduct/index.md

---
title: Code of conduct
description: The basic standards that we strive for across ethereum.org spaces.
lang: en
---

> ## Mission

Reviewer: ```suggestion
## Mission {#mission}
```

To develop and maintain the most comprehensive and accessible knowledge hub for Ethereum.

> ## Values

Reviewer: ```suggestion
## Values {#values}
```

The [Ethereum.org](http://Ethereum.org) community strives to be:

- educational, intended to help everyone to understand Ethereum
- inclusive
- accessible
- community-driven
- focused on Ethereum’s underlying technology and use-cases
- focused on Ethereum concepts and design principles

> ## What we are not

Reviewer: ```suggestion
## What we are not {#what-we-are-not}
```

- The Ethereum Foundation website
- A platform for promoting investments or profiteering of any kind
- A platform for elevating or endorsing individual projects or organizations
- A DEX, CEX or any other form of financial platform
- A platform that gives financial or legal advice of any kind

## Code-of-conduct

> ### Pledge

Reviewer: ```suggestion
### Pledge {#pledge}
```

> Open participation is core to the [ethereum.org](http://ethereum.org) ethos. We are a website and community maintained by thousands of contributors and this is only possible if we maintain a welcoming participatory environment. To this end, contributors to this site pledge to maintain a harassment-free environment for all participants across all ethereum.org platforms and community spaces. The ethereum.org community welcomes and values anyone who wants to participate in a constructive and friendly way, regardless of age, disability, ethnicity, sex characteristics, gender identity, level of experience, area of expertise, education, socio-economic status, nationality, personal appearance, race, religion or any other dimension of diversity.

Reviewer: ```suggestion
Open participation is core to the [ethereum.org](http://ethereum.org) ethos. We are a website and community maintained by thousands of contributors, and this is only possible if we maintain a welcoming, participatory environment. To this end, contributors to this site pledge to maintain a harassment-free environment for all participants across all ethereum.org platforms and community spaces. The ethereum.org community welcomes and values anyone who wants to participate in a constructive and friendly way, regardless of age, disability, ethnicity, sex characteristics, gender identity, level of experience, area of expertise, education, socio-economic status, nationality, personal appearance, race, religion or any other dimension of diversity.
```

> ### Scope

Reviewer: ```suggestion
### Scope {#scope}
```

> This Code of Conduct applies to all [ethereum.org](http://ethereum.org) spaces (such as Github, Discord, Figma Crowdin, Twitter and other online platforms), and it also applies when the community is represented in real-world public spaces such as at meetups, conferences and events.

Reviewer: ```suggestion
This Code of Conduct applies to all ethereum.org spaces (such as GitHub, Discord, Figma Crowdin, Twitter and other online platforms), and it also applies when the community is represented in real-world public spaces such as at meetups, conferences and events.
```

> ### Our standards

Reviewer: ```suggestion
### Our standards {#our-standards}
```

Examples of behaviour that contributes to creating a positive environment include:

- Using welcoming and inclusive language
- Being respectful of differing viewpoints and experiences
- Gracefully accepting and/or empathetically providing constructive criticism
- Acting calmly and professionally when resolving conflicts or disagreements
- Showing empathy and tolerance towards other community members
- Encouraging and amplifying new voices in the community

Examples of unacceptable behavior by participants include:

- Physical violence, threatening physical violence or encouraging of physical violence of any kind
- Using sexualized language or imagery or imposing unwelcome sexual attention
- Impersonating another individual or otherwise dishonestly claiming affiliation with some individual or organization.
- Trolling, insulting/derogatory comments, and personal or political attacks

> - Harrassing other community members in public or private channels

Reviewer: ```suggestion
- Harassing other community members in public or private channels
```

> - Publishing others’ private information, such as a physical or electronic addresses, without explicit permission

Reviewer: ```suggestion
- Publishing others’ private information, such as a physical or electronic address, without explicit permission
```

- Social engineering, scamming or otherwise manipulating other community members
- Promoting investments, tokens, projects or anything else for personal monetary or non-monetary gain
- Spamming servers with off-topic content
- Disregarding requests or warnings from community moderators
- Engaging in other conduct which could reasonably be considered inappropriate in a professional setting

> ### Reporting

Reviewer: ```suggestion
### Reporting {#reporting}
```

> Violations of the code of conduct will normally be visible to the community as we try to do everything in open, public channels, allowing community members to self police.

Reviewer: ```suggestion
Violations of the code of conduct will normally be visible to the community as we try to do everything in open, public channels, allowing community members to self-police.
```

However, if something happens that you feel needs attention, you can raise it with someone who has a moderation role (e.g. discord guide) so that they can help investigate and execute the appropriate response.

> When reporting, please try to include as much detail as you can, including specific examples and timestamps where possible. This will help to ensure a fair outcome.

Reviewer: ```suggestion
When reporting, please include as much detail as possible, including specific examples and timestamps. This will help to ensure a fair outcome.
```

> ### Enforcement

Reviewer: ```suggestion
### Enforcement {#enforcement}
```

> Depending on the severity, people who violate the code of conduct can receive warnings, temporary bans or permanent bans from [ethereum.org](http://ethereum.org) platforms.

Reviewer: ```suggestion
Depending on the severity, people who violate the code of conduct can receive warnings, temporary bans or permanent bans from ethereum.org communities.
```
